src/API/src/db/shared/entities/recordedSpecies.resolver.test.ts

```diff
@@ -0,0 +1,30 @@
+import { buildTestingModule } from '../../../testHelpers';
```

Comment:
Why is this test file here? It needs to be moved down a folder

Reply:
I just had a quick look in the docs as I was certain that that is where Ray had created it to begin with:

"We will create a central dependency injection module in a test helper, located in `src/API/src/testHelpers.ts` that provides a way of building mocked versions of the dependencies."

Is there a reason we would want it moved?

Reply:
Test files should always be at the same level as the file they're testing, unless there's a compelling reason not to do that.